packages/salesforcedx-vscode-core/test/integration/testdata/aura/PictureGalleryCard/PictureGalleryCard.cmp

```diff
@@ -0,0 +1,30 @@
+<aura:component implements="flexipage:availableForAllPageTypes"
```

Comment:
We should move these to a new folder/project under `system-tests/assets/`. That's where we already have a couple of test projects for scenarios where we need metadata.

Reply:
Will do.